solvable_files.sh

```diff
@@ -112,7 +112,7 @@ function correct_mtime() {
 
 	if [ "$action" == "fix" ]
 	then
-		touch -c "$filepath"
+		touch -c "$filepath" -m -d "$(date --date @"$mtime_in_db")"
```

Comment:
If we do that, then the `mtime` won't change as it is the same in the DB and on the FS. See: https://github.com/nextcloud-gmbh/mtime_fixer_tool_kit/blob/0851283c5fbfbb6c14c1b8f102ee70045e619a49/solvable_files.sh#L105

Did you have a situation where this would make sense?

Reply:
The question was how we could solve thie?

https://help.nextcloud.com/t/why-is-23-0-1-so-heavily-delayed/132133/28

Reply:
Ok, I understand the motivation, but as the `mtime` is lost on both sides, db and fs, then this will not help.

Reply:
The only option is to restore the `mtime` from a backup.